src/tools/run-make-support/src/diff/tests.rs

```diff
@@ -0,0 +1,38 @@
+#[cfg(test)]
+mod tests {
```

Comment:
I think the tests here will not be executed in CI for now (I don't know how to configure it).

Reply:
I think that is fine for now, but if you want to add support for running the unit tests in `run-make-support`, you'll need to do something like `CompiletestTest`:

https://github.com/rust-lang/rust/blob/7f2fc33da6633f5a764ddc263c769b6b2873d167/src/bootstrap/src/core/build_steps/test.rs#L731

Modulo the unstable test feature part because `run-make-support` is stable-only and does not use any unstable (cargo or rustc) features.

Reply:
I filed issue https://github.com/rust-lang/rust/issues/124267 to track this.